bpf/stbtrace/io.st

```diff
@@ -128,7 +128,6 @@ int disk_io_done(struct pt_regs *ctx, struct request *reqp)
 """
 b = BPF(text=bpf_text)
 
-b.attach_kprobe(event="blk_start_request", fn_name="disk_io_start")
```

Comment:
As I mentioned in today's meeting, assuming that this was needed on 4.15 kernel, because we're using the 4.15 kernel in AWS, we'll need to selectively use this probe still.

On a slightly related note, did this function get removed and get subsumed by `blk_mq_start_request`, or get renamed? Just making sure that we're not missing some other I/O path by removing this probe.

Reply:
So `blk_start_request` was removed in this cleanup - [block: remove dead elevator code](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/block/blk-core.c?id=a1ce35fa49852db60fc6e268038530be533c5b15)
And looking at the 5.x code, `blk_mq_start_request` seems to cover all I/O paths, also 
here is a [bcc reference](https://github.com/iovisor/bcc/pull/2070#issuecomment-445772909) on the same.